docs/Server-Setup.md

```diff
@@ -83,6 +83,17 @@ Start the CodaLab service as follows:
 
     ./codalab_service.py start -bd
 
+If your default python version is not python 3.6+, then you might have to use:
+
+    python3 codalab_service.py start -bd
+
+For macOS, you might come across an error with `gunicorn` when running the above command.
```

Comment:
Maybe this should be put in a section at the end called Troubleshooting so the main instructions don't get too long.